src/pages/questionPage.js

```diff
@@ -40,31 +40,49 @@ if(!isSelectedOneAnswer){
 }
 // Show user if user selection correct or not
 const showAnswerIsCorrect = (isAnswerCorrect,target) => {
-  isAnswerCorrect ? setBackgroundColor('green',target) : setBackgroundColor('red',target);
+  if(isAnswerCorrect){
+    setBackgroundColor('green',target)
+  }
+  else{
+    setBackgroundColor('red',target); 
+    showCorrect()
+
+  }
 }
 
 const setBackgroundColor = (color,target)=> {
   target.style.backgroundColor = color;
 }  
-
+const showCorrect =()=> {
+  const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+  const answersListElement = document.getElementById(ANSWERS_LIST_ID);
+  const options= answersListElement.querySelectorAll('li')
+ return options.forEach((op)=>
```

Comment:
It should not be needed to get all options and check each of them to see if contains the correct answer, since we know already which element has the correct answer ( question[answers][questions.correct] ).